src/redux/rootReducer.ts

```diff
@@ -1,9 +1,11 @@
 import { Action, combineReducers } from '@reduxjs/toolkit';
 import { appVersionReducer } from './features/appVersion/appVersionSlice';
+import { observationsResultsReducer } from './features/observations/observationsSlice';
 
 // assembly of app reducers
 export const reducers = {
   appVersion: appVersionReducer,
+  observationsResults: observationsResultsReducer,
```

Comment:
@charlie-tf the `state.observationResults` comes from this key

Reply:
Ohh, gotcha 👍